engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/scenariotesting/TestFlinkProcessCompilerDataFactory.scala

```diff
@@ -194,16 +189,14 @@ class TestFlinkProcessCompilerDataFactory(
 
     override def transformOriginalInvocationResult(
         originalSource: Any,
-        originalSourceWasWrappedInContextTransformation: Boolean,
+        outputValidationContext: ValidationContext,
         typingResult: TypingResult,
-        compilationDependencies: NodeCompilationDependencies,
-        invocationContext: Option[ComponentImplementationSpecificInvocationContext]
+        compilationDependencies: NodeCompilationDependencies
     ): Any = {
-      val outputValidationContext =
-        determineOutputValidationContext(originalSource, compilationDependencies, invocationContext)
-
       val commonFormatRecordsNelOpt =
```

Comment:
this nel is build just to match on if its empty. This can be simplified to an `if` statement with `.isEmpty` check